server/build/plugins/combine-assets-plugin.js

```diff
@@ -9,19 +11,27 @@ export default class CombineAssetsPlugin {
 
   apply (compiler) {
     compiler.plugin('after-compile', (compilation, callback) => {
-      let newSource = ''
+      const concat = new Concat(true, this.output, '\n')
+
       this.input.forEach((name) => {
         const asset = compilation.assets[name]
+        const sourceMap = compilation.assets[`${name}.map`]
         if (!asset) return
 
-        newSource += `${asset.source()}\n`
-
-        // We keep existing assets since that helps when analyzing the bundle
+        concat.add(name, asset.source(), sourceMap && sourceMap.source())
+        delete compilation.assets[name]
```

Comment:
What is the debugging flow for using these? I didn't find these files all that useful since they themselves are transformed from other content, but I may have missed their utility.